common/nodejs-utils/src/download_file.rs

```diff
@@ -0,0 +1,106 @@
+use reqwest::IntoUrl;
+use reqwest_middleware::ClientBuilder;
+use reqwest_retry::policies::ExponentialBackoff;
+use reqwest_retry::RetryTransientMiddleware;
+use std::fs::File;
+use std::io;
+use std::path::{Path, PathBuf};
+use std::time::Duration;
+
+const DEFAULT_CONNECT_TIMEOUT: Duration = Duration::from_secs(5);
+
+const DEFAULT_READ_TIMEOUT: Duration = Duration::from_secs(10);
+
+const DEFAULT_RETRIES: u32 = 5;
```

Comment:
Comment: As an FYI I dug into the underlying code and "5 max retries" means "5 max sleeps" but actually 6 requests. Not a problem, just thought it was worth sharing.